src/Cli/dotnet/Commands/New/NewCommandParser.cs

```diff
@@ -170,7 +168,7 @@ private static CliTemplateEngineHost CreateHost(
             { "prefs:language", preferredLang },
             { "dotnet-cli-version", Product.Version },
             { "RuntimeFrameworkVersion", new Muxer().SharedFxVersion },
-            { "NetStandardImplicitPackageVersion", new FrameworkDependencyFile().GetNetStandardLibraryVersion() },
+            { "NetStandardImplicitPackageVersion", new FrameworkDependencyFile().GetNetStandardLibraryVersion() ?? "" },
```

Comment:
@baronfel should verify this change as being accurate. It is also possible the templating engine is wrong and it should accept a `Dictionary<string, string?>` but I don't have the depth to say which is better.

Reply:
The `GetNetStandardLibraryVersion` method shouldn't return a nullable - it's one of that hard-coded, required things and we should blow up if it's not present.

Reply:
Happy to change that to throw on failure if you think it's the best course.